src/odemis/driver/actuator.py

```diff
@@ -1307,13 +1309,14 @@ def __init__(self, name, role, dependencies, axis_name, positions, cycle=None,
 
         self._cycle = cycle
         self._move_sum = 0
-        self._position = {}
-        self._referenced = {}
+        self._position: Dict[str, float] = {}
+        self._referenced: Dict[str, bool] = {}
         axis, dep = list(dependencies.items())[0]
         self._axis = axis
         self._dependency = dep
         self._caxis = axis_name
-        self._positions = positions
+        self._positions: Union[Set[float], Dict[float, str]] = positions
+        self._actual_positions: Dict[float, float] = {}  # one choice position -> dependency position
```

Comment:
Let's add a comment above this variable to describe what it is. Something like "Latest position reached after move to given position was requested".